pkg/acs/calacs/acsccd/doccd.c

```diff
@@ -32,6 +35,8 @@ static void dqiMsg (ACSInfo *, int);
 static void BiasMsg (ACSInfo *, int);
 static void BlevMsg (ACSInfo *, int);
 static void SinkMsg (ACSInfo *, int);
+/* These are the subarray supported apertures for the bias shift correction */
+static const char *subApertures[] = {"WFC1A-2K", "WFC1B-2K", "WFC2C-2K", "WFC2D-2K"};
```

Comment:
If you ever want to add polarizers, keep in mind that the aperture name never changed after flight software update, so same POL aperture is "old subarray" before that update, and "new subarray" after. Becareful with those.